systemuser.sh

```diff
@@ -28,7 +32,7 @@ JPY_LOCAL_DIR=$SCRATCH_HOME/.local
 mkdir -p $JPY_LOCAL_DIR
 export JUPYTER_CONFIG_DIR=$JPY_DIR
 # Our kernels will be in $JPY_LOCAL_DIR/share/jupyter, $LCG_VIEW/share/jupyter is needed for the notebook extensions
-export JUPYTER_PATH=$JPY_LOCAL_DIR/share/jupyter:$LCG_VIEW/share/jupyter
+export JUPYTER_PATH=$JPY_LOCAL_DIR/share/jupyter:$LCG_VIEW/share/jupyter$EXTRA_LIBS
```

Comment:
$EXTRA_LIBS is a path pointing to a folder that, in the future, will have SWAN nbextensions. Like the LCG ones. 
I've put the ':' inside $EXTRA_LIBS, yes. I will move them here.

Reply:
Ok thanks for the explanation. However, since this PR (and commit) is about the extra save, I believe this particular change should not belong to this PR.

Reply:
Do you want me to make a new PR with just this line? I joined these two things because this PR is not just for "extra save", is for "v2.11" (that contains extra save + notifications nbextensions).

Reply:
I would at least divide it in two commits, each with its own message :smile: 
I don't know if you can edit now the description of the PR, if possible please introduce some text there as well, thanks!